internal/telemetry/Tracker.go

```diff
@@ -0,0 +1,37 @@
+package telemetry
+
+import (
+	"fmt"
+)
+
+// Tracker logs events
+type Tracker interface {
+	Track(event *Event)
```

Comment:
curious as to why this is a pointer to Event here?

Reply:
I wasn't sure if we considered an event to be critically large enough to pass by reference instead of value.  Should I not make it a pointer?

Reply:
yeah, no pointer needed here